.github/workflows/api-test.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         uses: actions/cache@v4
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}-api-test
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}-backend
```

Comment:
Why change the cache key?

Reply:
The total cache storage limit is 10GB. We add binary package to it will cause overuse. So I reuse it. Please refer to the doc in 
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy